robottelo/manifests.py

```diff
@@ -119,12 +125,13 @@ class Manifest(object):
         with Manifest() as manifest:
             # my fancy stuff
     """
-    def __init__(self, content=None, filename=None):
+    def __init__(self, content=None, filename=None, manifest='default'):
         self._content = content
         self.filename = filename
+        self.manifest = manifest
 
         if self._content is None:
-            self._content = _manifest_cloner.clone()
+            self._content = _manifest_cloner[self.manifest].clone()
```

Comment:
Doing as I suggested on the previous comment this call would be something like `_manifest_cloner.clone(self.manifest)`